go.mod

```diff
@@ -18,6 +20,7 @@ require (
 	github.com/openshift/api v0.0.0-20211209135129-c58d9f695577
 	github.com/openshift/build-machinery-go v0.0.0-20210806203541-4ea9b6da3a37
 	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
+	github.com/openshift/gssapi v0.0.0-20161010215902-5fb4217df13b
```

Comment:
This library requires static bindings to gssapi library which are one of the biggest pains in oc preventing us from upgrading rhel or cross-building binaries for OS-es other than linux. Is there any potential danger for other consumers?